js-module-base/pom.xml

```diff
@@ -70,7 +72,7 @@
             <id>copy-jsmodules</id>
             <activation>
                 <file>
-                    <exists>${project.basedir}/src/main/webapp/jsmodules</exists>
+                    <exists>${basedir}/src/main/webapp/jsmodules</exists>
```

Comment:
ditto

Reply:
Yes, that's what I originally had but it failed and maven (3.3.9 in my case) logged a warning instructing to change to use just "basedir".

I'll go back and reproduce the error and see if it's a version thing etc.